doc/ref/configuration/minion.rst

```diff
@@ -81,9 +81,10 @@ The option can can also be set to a list of masters, enabling
 ``ipv6``
 --------
 
-Default: ``False``
+Default: ``None``
```

Comment:
The default behavior is all-IPv4 for salt master and salt minion, we don't change that. This patch doesn't change behavior for this case. For people who run IPv6-only deployments already, this patch doesn't change behavior either, because those people already have to set `ipv6: true` on both minion and master.

The point here is to enable new behavior out of the box, not change anything existing and working.

This patch only changes behavior for the case that was not possible before, so it's hard to break anything existing in the wild. The only thing I can think of is a deployment where salt master works on IPv4, listens and accepts connections on IPv6, but otherwise breaks on IPv6. You have to try very hard to achieve that and I double anyone has would have this issue.

In fact, `ipv6` option wasn't even documented, I changed that recently in #39131.

If you still feel very strongly about development branch, I can do that.